shell/initialize/entry-helpers.js

```diff
@@ -199,14 +197,6 @@ async function render(to, from, next) {
 
   // If no Components matched, generate 404
   if (!Components.length) {
-    // Call the authenticated middleware. This used to attempt to load the error layout but because it was missing it would:
-    // 1. load the default layout instead
-    // 2. then call the authenticated middleware
-    // 3. Authenticated middleware would then load plugins and check to see if there was a valid route and navigate to that if it existed
-    // 4. This would allow harvester cluster pages to load on page reload
-    // We should really make authenticated middleware do less...
-    await callMiddleware.call(this, [{ options: { middleware: ['authenticated'] } }], app.context);
```

Comment:
This could cause problems with loading harvester cluster pages. I haven't been able to test yet.

Reply:
I think we should do that, so we do not risk rollback.
@torchiaf is it something we can try?